rpyc/rpyc_client/command_handlers.py

```diff
@@ -0,0 +1,203 @@
+import psutil
+import datetime
+
+"""
+All the command handlers.
+They parse, execute and possibly format the output for the terminal.
+They (must) also return (and receive) additional information to the command_handlers_manager. 
+"""
+
+
+class CommandHandler(object):
+    def __init__(self, rpyc_conn):
+        self.rpyc_conn = rpyc_conn
+        self.empty_output = (None, None)
+
+    def _parse_input(self, split_input, additional_input):
+        pass
+
+    def _execute(self, **kwargs):
+        return self.empty_output
+
+    def _format_output(self, output):
+        return output
+
+    def handle_command(self, split_input, additional_input):
+        try:
+            kwargs = self._parse_input(split_input, additional_input)
+            terminal_output, manager_output = self._execute(**kwargs)
+            terminal_output = self._format_output(terminal_output) if terminal_output else terminal_output
```

Comment:
same note for the format output function
it is good that you divide to multiple functions, but they can all be called from the _execute func